agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java

```diff
@@ -41,14 +32,26 @@ public String getAllRowsForGeneDiseaseAnnotations(List<GeneDiseaseAnnotationDocu
 			new DownloadHeader<>("Species Name", (DiseaseDownloadRow::getSpeciesName)),
 			new DownloadHeader<>("Gene ID", (DiseaseDownloadRow::getMainEntityID)),
 			new DownloadHeader<>("Gene Symbol", (DiseaseDownloadRow::getMainEntitySymbol)),
+			new DownloadHeader<>("Additional Implicated Gene ID", (DiseaseDownloadRow::getAdditionalImplicatedGeneSymbols)),
+			new DownloadHeader<>("Additional Implicated Gene Symbol", (DiseaseDownloadRow::getAdditionalImplicatedGeneIds)),
 			new DownloadHeader<>("Genetic Entity ID", (DiseaseDownloadRow::getGeneticEntityID)),
 			new DownloadHeader<>("Genetic Entity Name", (DiseaseDownloadRow::getGeneticEntityName)),
 			new DownloadHeader<>("Genetic Entity Type", (DiseaseDownloadRow::getGeneticEntityType)),
 			new DownloadHeader<>("Association", (DiseaseDownloadRow::getAssociation)),
+			new DownloadHeader<>("Disease Qualifier", (DiseaseDownloadRow::getDiseaseQualifier)),
```

Comment:
I think this should be "Disease Qualifiers"

Reply:
In the mockup it states 'Disease Qualifier'